modules/node/handler/node_page_builder.e

```diff
@@ -0,0 +1,44 @@
+note
+	description: "Paginator builder for CMS nodes."
+	date: "$Date$"
+	revision: "$Revision$"
+
+class
+	NODE_PAGE_BUILDER
+
+inherit
+
+	NODE_PAGINATION_BUILDER [CMS_NODE]
+
+	CMS_NODE_HANDLER
```

Comment:
The purpose is to get access to node_api and api, at the moment only node_api is used.